src/Nancy.Demo.Authentication.Stateless.Website/Web.config

```diff
@@ -1,7 +1,19 @@
 ﻿<?xml version="1.0"?>
-
+<!--
+  For more information on how to configure your ASP.NET application, please visit
+  http://go.microsoft.com/fwlink/?LinkId=169433
+  -->
 <configuration>
+  <!--

```

Comment:
I'm guessing these comments came from the .NET 4.5 update commits that shouldn't be here? Otherwise we should kill these.